src/public/js/startData.js

```diff
@@ -0,0 +1,79 @@
+
+export const startData = class StartData {
+
+    getPlayersName() {
+
+        const playersName = []
+        let name
+        do {
+            name = prompt('Введите имя игрока. Если вы ввели имена всех игроков, то, чтобы продолжить нажмите ОК')
+            if (name!= ''){
```

Comment:
```suggestion
            if (name != ''){
```
всегда разделяй знаки операций и сравнения пробелом с обеих сторон

Reply:
есть